controllers/machinedeploymentupgrade_controller.go

```diff
@@ -43,12 +56,63 @@ func NewMachineDeploymentUpgradeReconciler(client client.Client) *MachineDeploym
 //+kubebuilder:rbac:groups=anywhere.eks.amazonaws.com,resources=machinedeploymentupgrades/finalizers,verbs=update
 
 // Reconcile reconciles a MachineDeploymentUpgrade object.
-func (r *MachineDeploymentUpgradeReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	_ = log.FromContext(ctx)
+// nolint:gocyclo
```

Comment:
What errors do we see if we remove this?

Reply:
Things have moved around so we don't need this anymore. Removed it

Reply:
`cyclomatic complexity 12 of func `(*MachineDeploymentUpgradeReconciler).Reconcile` is high `

Reply:
Can we link the GitHub issue like we do here? But this doesn't have to block the merge of the PR: https://github.com/aws/eks-anywhere/blob/main/controllers/cluster_controller.go#L205